### PR TITLE
Feature/us11062 landing page

### PR DESCRIPTION
**Pull request by zarahzachz**
Create landing page for SRFP. Designs should match mockups [here](https://app.zeplin.io/project/5a284b164a399c047124d67e/screen/5a284b62b1549395c7870e0a).

No corresponding PRs.

### CODE REVIEW

File: src/app/components/home/home.component.html [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
-<app-fire-content-block id="abc123"></app-fire-content-block>
-<br>
-<app-fire-content-block id="pjl1"></app-fire-content-block>
-<br>
-<app-fire-content-block id="dream9"></app-fire-content-block>
+<div class="jumbotron jumbotron-xl" style="background-image: url('./assets/map.svg');">
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12 col-md-8 col-md-offset-2">
+        <app-fire-content-block id="HomepageJumbotronText"></app-fire-content-block>
```
Comment (galiotoweb) [inline src/app/components/home/home.component.html]: The styling for the paragraphs in this section is a little off. In Zeplin it's Acumin Pro Light, with font-size: 19px; line-height: 27px; opacity: 0.8;

File: src/app/components/home/home.component.html [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12 col-md-8 col-md-offset-2 push-ends soft-ends">
+      <section class="font-size-large text-center">
+        <app-fire-content-block id="HomepageText"></app-fire-content-block>
```
Comment (galiotoweb) [inline src/app/components/home/home.component.html]: The styling in this section should also be Acumin Pro Light, with font-size: 19px; line-height: 27px; but the opacity is 1.

**Comment by zarahzachz** [comment]
@galiotoweb I adjusted the font size and opacity of the jumbotron text. 

Ready for review.

**Comment by zarahzachz** [comment]
@seancdavis I updated the font-size on small screens.

Ready for review.

**Comment by zarahzachz** [comment]
@seancdavis I've gone ahead and made the adjustment to get the circle background on the social icons in the footer. If we decide to pass on this, it'll be easy to revert back, but it wasn't that much extra effort to get it to match.

**Pull request merged**